src/pagination/icons.js

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import React from 'react';
+
+/**
+ * Ideally the content of this file should be replaced with official icons when they are available
+ */
+
+export function ArrowLeft() {
+  return (
+    <svg viewBox="0 0 6 10" fill="none" style={{width: '6px', height: '10px'}}>
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M0 5.00001C3.57629e-07 5.26523 0.105358 5.51958 0.292894 5.70712L4.29289 9.70711C4.68342 10.0976 5.31658 10.0976 5.70711 9.70711C6.09763 9.31658 6.09763 8.68342 5.70711 8.29289L2.41421 5.00001L5.70711 1.70711C6.09763 1.31658 6.09763 0.683416 5.70711 0.292892C5.31658 -0.0976315 4.68342 -0.0976306 4.29289 0.292894L0.292892 4.29291C0.105356 4.48044 -3.57627e-07 4.7348 0 5.00001Z"
+        fill="#0490FF"
```

Comment:
It should probably have a `currentColor` value. Currently the color used is different from the button's text color.